paddle/framework/pybind.cc

```diff
@@ -42,6 +42,7 @@ USE_OP(fill_zeros_like);
 USE_OP_ITSELF(recurrent_op);
 USE_OP(gaussian_random);
 USE_OP(uniform_random);
+USE_CPU_ONLY_OP(gather);
```

Comment:
GPU test and cuda context not very ready. Easy to add it later, maybe next time :)